sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -377,7 +378,9 @@ private synchronized void loginFromUI() {
                 @Override
                 public void onSuccess(IAuthenticationResult authenticationResult) {
                     AppCenterLog.info(LOG_TAG, "User login succeeded. id=" + authenticationResult.getIdToken());
-                    // TODO send id token (not access token) to ingestion, if sdk still enabled.
+                    if (Identity.isEnabled().get()) {
```

Comment:
What thread is that since we are calling `.get`? Prefer using `thenAccept`.
Also do we need to fully qualify the static call since we are already in the same class?